### PR TITLE
Use the latest versions of hydra-head

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: spec/controllers/downloads_controller_spec.rb [hunk 1]
```diff
@@ -5,10 +5,10 @@
   describe "with a file" do
     let(:depositor) { FactoryGirl.find_or_create(:archivist) }
     let(:file) do
-      GenericFile.new.tap do |f|
+      GenericFile.create do |f|
         f.apply_depositor_metadata(depositor.user_key)
-        f.add_file(File.open(fixture_path + '/world.png'), 'content', 'world.png')
-        f.save!
+        f.label = 'world.png'
```
Comment (mjgiarlo) [inline spec/controllers/downloads_controller_spec.rb]: Can you say more about this addition?

Comment (mjgiarlo) [inline spec/controllers/downloads_controller_spec.rb]: We used to have a method that took care of setting the label and a default title, and I see the actor now does this itself in create_content. (And is "label" even a thing in Fedora 4?  I thought was a Fedora 3 built-in.)

Comment (jcoyne) [inline spec/controllers/downloads_controller_spec.rb]: Label was a fedora 3 builtin. We've replaced it with a simple predicate. `add_file` in hydra-head used to set that label, but we've deprecated that module so now we're doing it manually.

Comment (mjgiarlo) [inline spec/controllers/downloads_controller_spec.rb]: :+1:

File: app/jobs/ingest_local_file_job.rb [hunk 2]
```diff
@@ -20,12 +20,13 @@ def run
     path = File.join(directory, filename)
 
     generic_file.label = File.basename(filename)
-    generic_file.add_file(File.open(path), 'content', generic_file.label)
+    generic_file.add_file(File.open(path), path: 'content', original_name: generic_file.label, mime_type: mime_type(filename))
     generic_file.record_version_committer(user)
     generic_file.save!
```
Comment (mjgiarlo) [inline app/jobs/ingest_local_file_job.rb]: Should we be using Actor#create_content here?

Comment (jcoyne) [inline app/jobs/ingest_local_file_job.rb]: Yes we should (See https://github.com/projecthydra/sufia/blob/latest_hydra_head/app/jobs/ingest_local_file_job.rb#L15)  I took a stab at that, but I was getting too many unrelated issues, so I backed it out.

Comment (mjgiarlo) [inline app/jobs/ingest_local_file_job.rb]: OK. I'll merge this and put in an issue.

**Pull request merged**